### ISSUE TITLE
Fails to build on Windows

### ISSUE THREAD

**Issue by josh-hemphill** [caddyserver/xcaddy#57]
I initially ran `xcaddy build --output bin/caddy.exe --with github.com/caddyserver/nginx-adapter`
and it immediately throws an exception with a stack trace and blocks my terminal.

I've also tried with various other plugins, and all have the same result.

go version: go1.16.2 windows/amd64

I realize this isn't a lot of information, I'm not very familiar with Go; if you can direct me to how to further debug this, I'd be happy to.

<details>
  <summary>Stack Trace</summary>
  
  ```go
Exception 0xc0000005 0x0 0x7ffa55310fff 0x209cbd10000
PC=0x209cbd10000

runtime: unknown pc 0x209cbd10000
stack: frame={sp:0x81fcdfe650, fp:0x0} stack=[0x0,0x81fcdffd30)
00000081fcdfe550:  00000081fcdfe598  00000081fcdfe5c0
00000081fcdfe560:  00000081fcdfe588  00000081fcdfe580
00000081fcdfe570:  00000081fcdfe584  00000209a6990fc0
00000081fcdfe580:  0000000000000000  0000000000000000
00000081fcdfe590:  0000000000000000  0000000000000005
00000081fcdfe5a0:  00000081fcdfe6e8  00007ffa52acb3df
00000081fcdfe5b0:  00000209a67c3b40  00007ffa5514345e
00000081fcdfe5c0:  00007ffa551100e8  00000209a67f86f0
00000081fcdfe5d0:  00007ffa52acb3d9  00000081fcdfe620
00000081fcdfe5e0:  004f0044004e0049  0053005c00530057
00000081fcdfe5f0:  00000209a67c3b40  0000000000000000
00000081fcdfe600:  00000209a67dfc60  006c006400050005
00000081fcdfe610:  00007ffa52acb3d9  0000000000000000
00000081fcdfe620:  00007ffa00000000  00007ffa551100e8
00000081fcdfe630:  0000000000000000  0000000000000000
00000081fcdfe640:  0000000000000001  00007ffa55142513
00000081fcdfe650: <0000020900000001  0000000000000000
00000081fcdfe660:  0000000000000000  00000081fcdfe758
00000081fcdfe670:  0000000000000000  0000000000000000
00000081fcdfe680:  0000000000000000  0000000000000000
00000081fcdfe690:  00000209a67f86f0  00007ffa55261180
00000081fcdfe6a0:  00000209a67c3b40  00007ffa552637a0
00000081fcdfe6b0:  000000000000097d  00007ffa51ccccd8
00000081fcdfe6c0:  00007ffa51cca148  00007ffa552611a8
00000081fcdfe6d0:  00007ffa55273fcf  00007ffa51cc0000
00000081fcdfe6e0:  00007ffa55265d94  00007ffa55164c60
00000081fcdfe6f0:  0000000000000000  0000000000000000
00000081fcdfe700:  0000000000000000  0000000000000000
00000081fcdfe710:  00000209a67f3d80  0000000000000044
00000081fcdfe720:  0000000000000003  00007ffa5527a3f0
00000081fcdfe730:  0000000000000001  00000081fcdfea00
00000081fcdfe740:  00000209a67dfc60  00007ffa5516eb58
runtime: unknown pc 0x209cbd10000
stack: frame={sp:0x81fcdfe650, fp:0x0} stack=[0x0,0x81fcdffd30)
00000081fcdfe550:  00000081fcdfe598  00000081fcdfe5c0
00000081fcdfe560:  00000081fcdfe588  00000081fcdfe580
00000081fcdfe570:  00000081fcdfe584  00000209a6990fc0
00000081fcdfe580:  0000000000000000  0000000000000000
00000081fcdfe590:  0000000000000000  0000000000000005
00000081fcdfe5a0:  00000081fcdfe6e8  00007ffa52acb3df
00000081fcdfe5b0:  00000209a67c3b40  00007ffa5514345e
00000081fcdfe5c0:  00007ffa551100e8  00000209a67f86f0
00000081fcdfe5d0:  00007ffa52acb3d9  00000081fcdfe620
00000081fcdfe5e0:  004f0044004e0049  0053005c00530057
00000081fcdfe5f0:  00000209a67c3b40  0000000000000000
00000081fcdfe600:  00000209a67dfc60  006c006400050005
00000081fcdfe610:  00007ffa52acb3d9  0000000000000000
00000081fcdfe620:  00007ffa00000000  00007ffa551100e8
00000081fcdfe630:  0000000000000000  0000000000000000
00000081fcdfe640:  0000000000000001  00007ffa55142513
00000081fcdfe650: <0000020900000001  0000000000000000
00000081fcdfe660:  0000000000000000  00000081fcdfe758
00000081fcdfe670:  0000000000000000  0000000000000000
00000081fcdfe680:  0000000000000000  0000000000000000
00000081fcdfe690:  00000209a67f86f0  00007ffa55261180
00000081fcdfe6a0:  00000209a67c3b40  00007ffa552637a0
00000081fcdfe6b0:  000000000000097d  00007ffa51ccccd8
00000081fcdfe6c0:  00007ffa51cca148  00007ffa552611a8
00000081fcdfe6d0:  00007ffa55273fcf  00007ffa51cc0000
00000081fcdfe6e0:  00007ffa55265d94  00007ffa55164c60
00000081fcdfe6f0:  0000000000000000  0000000000000000
00000081fcdfe700:  0000000000000000  0000000000000000
00000081fcdfe710:  00000209a67f3d80  0000000000000044
00000081fcdfe720:  0000000000000003  00007ffa5527a3f0
00000081fcdfe730:  0000000000000001  00000081fcdfea00
00000081fcdfe740:  00000209a67dfc60  00007ffa5516eb58
rax     0x7ffa51ccd87c
rbx     0x7ffa51ccd87a
rcx     0x41
rdi     0xffffffffffbadd11
rsi     0x0
rbp     0x7ffa51e1a100
rsp     0x81fcdfe650
r8      0x0
r9      0x0
r10     0x0
r11     0x97c
r12     0xc000007a
r13     0x0
r14     0x7ffa51ccd87c
r15     0x7ffa55110000
rip     0x209cbd10000
rflags  0x10202
cs      0x33
fs      0x53
gs      0x2b
  ```
  
</details>

**Comment by francislavoie** [comment]
How did you build/download `xcaddy`? What version of Go do you have installed (run `go version`)?

**Comment by josh-hemphill** [comment]
It might be an issue with the GoLang installation. I had tried installing Go with chocolatey, and got the error when trying to build. For some reason it just started working after I built xcaddy from source after reinstalling Go with Go's `msi` installer.  I think I got the error again after the reinstall because I had put the xcaddy binary I had built previously in my user `bin` folder and it carried over after the reinstall. I would have thought that building xcaddy would have failed if building _with_ xcaddy ultimately failse.
So I guess that makes it an issue with installing Go with chocolatey on Windows. It didn't look like it was an official installation method, should I open an issue with a GoLang repository?
